worker/worker/request_handlers.go

```diff
@@ -577,18 +577,12 @@ func extractKeywords(ctx *StreamContext) error {
 	}
 
 	// extract keywords
-	engExtractor := ocr.NewOcrExtractor(files, []string{"eng"})
+	engExtractor := ocr.NewOcrExtractor(files, []string{"eng", "deu"})
```

Comment:
IMO extractor would be a better name now

Reply:
yup, definitely. 👍🏻